confdgnmi/testtool/README.md

```diff
@@ -0,0 +1,122 @@
+# gNMI Telemetry testing tool
+
+This project is intended as a tool to help with automation testing of a telemetry capabilities of target device.
+It uses a [Robot Framework](https://robotframework.org/) (a generic open source automation framework) to implement range of test suites and
+test cases. These try to identify capabilities of a device to service typical telemetry related gNMI requests.
+
+## requirements
+
+We strongly depend on **Python3** for running the tests. All the `python` commands in following text assume that this binary resolves to a (fairly recent) python3 interpreter. Following is an example of supported (not necessarily lowest required) version:
+
+```bash
+$ python --version
+Python 3.8.10
+```
+
+See Robot's [installation instructions](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#installation-instructions) for details on how to install the framework. To verify you have a `robot` installed, you can run `robot --version` shell command. As of writing this document, current version is:
+
+```bash
+$ robot --version
+Robot Framework 6.0.2 (Python 3.8.10 on linux)
+```
+
+Actual test-runs require that supplementary gNMI client code (located in `../src`, primarily `confd_gnmi_client.py` module) can be run successfully. See the `../README.md` for requirements if you run into some issues.
```

Comment:
The README file is just a pointer to the .adoc document, and that describes a lot more than just how to generate the stubs and set the Python path, which is everything you need to run the client.

Reply:
will add clarifying comment...